Mathlib/Analysis/LocallyConvex/Bounded.lean

```diff
@@ -174,7 +174,7 @@ lemma isVonNBounded_iff_tendsto_smallSets_nhds {𝕜 E : Type*} [NormedDivisionR
 alias ⟨IsVonNBounded.tendsto_smallSets_nhds, _⟩ := isVonNBounded_iff_tendsto_smallSets_nhds
 
 lemma isVonNBounded_iff_absorbing_le {𝕜 E : Type*} [NormedDivisionRing 𝕜]
-    [AddCommGroup E] [Module 𝕜 E] [TopologicalSpace E] {S : Set E} :
+    [Zero E] [MulAction 𝕜 E] [TopologicalSpace E] {S : Set E} :
```

Comment:
I think this is spurious. `IsVonNBounded` is only relevant in topological vector spaces.